mod/developers/views/default/forms/developers/settings.php

```diff
@@ -22,6 +22,7 @@
 			'value' => $info['value'],
 			'checked' => $info['checked'],
 			'class' => $class,
+			'disabled' => $info['disabled'],
```

Comment:
rewrite this to elgg_view_field an you do not need all the disabled stuff... or use an inline if or something